virt-who.conf

```diff
@@ -90,5 +90,5 @@ VIRTWHO_DEBUG=0
 # Options for Kubevirt mode
 #VIRTWHO_KUBEVIRT_OWNER=
 #VIRTWHO_KUBEVIRT_ENV=
-#KUBECONFIG=
+#VIRTWHO_KUBEVIRT_KUBECONFIG=
 
```

Comment:
This is the name of an environment variable. Shouldn't it be 'kubeconfig'?

Reply:
We are not using environment variable anymore. We rely on command line or config file now.

Reply:
then remove this line from this file.

Reply:
Are you aware that you can specify a single config file on the command line, using '-c'?

Reply:
I did not know this option. Thanks for a hint.

Reply:
Does that reduce the necessity of this change?

Reply:
No, currently kubevirt plugin won't use this property from config file at all. This PR makes it possible.